static/app/components/autoComplete.tsx

```diff
@@ -390,14 +392,14 @@ class AutoComplete<T extends Item> extends React.Component<Props<T>, State<T>> {
     };
   };
 
-  getMenuProps = <E extends Element>(props?: GetMenuArgs<E>): GetMenuArgs<E> => {
+  getMenuProps<E extends Element>(props?: GetMenuArgs<E>): GetMenuArgs<E> {
     this.itemCount = props?.itemCount;
 
     return {
       ...(props ?? {}),
-      onMouseDown: this.handleMenuMouseDown,
+      onMouseDown: this.handleMenuMouseDown.bind(this),
```

Comment:
I usually prefer `=>` over bind since the cost of doing an arrow function above is basically negligible.